html-practices/my-new-change/script.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
```

Comment:
**chore** 🧹
Please set a comment to explain the purpose of this file

<sub><sup>I'm using [conventional comments](https://conventionalcomments.org/) formatting to improve comments readability and reader's expectations in this PR.</sup></sub>